PeraWallet/Classes/Utils/Extensions/Foundation/URL/URL+Extensions.swift

```diff
@@ -26,6 +26,10 @@ extension URL {
         return scheme?.lowercased() == "mailto"
     }
     var isWebURL: Bool {
+        // vestige.fi loads iframe src as blob:https://vestige.fi/... so we need to handle that case
+        if scheme?.lowercased() == "blob", let newURL = URL(string: String(absoluteString.suffix(absoluteString.count - 5))) {
```

Comment:
This line will crash if `absoluteString.count` will be less than 5. Consider using `absoluteString.dropLast(5)` instead.

Reply:
It's better be safe than sorry.